scripts/seed-data

```diff
@@ -13,5 +13,5 @@ fi
 mkdir -p ${BASE_DIR}/lib/rancher/conf
 
 for yaml in "${CLOUD_DATA}" '/scripts/conf/rancher.yml';do 
-    cp $yaml ${BASE_DIR}/lib/rancher/conf/
+    cp $yaml ${BASE_DIR}/lib/rancher/conf/rancher.yml
```

Comment:
I copy two here. I set a rancher.yml and a user defined file. The rancher.yml in the current usecase tells cloud-init to look for user data at that file location. (this is how the vagrant insecure key for instance is getting added.)


This needs to stay as it is.